plasmapy/formulary/quantum.py

```diff
@@ -375,10 +371,14 @@ def Wigner_Seitz_radius(n: u.m**-3):
 
     """
     radius = (3 / (4 * np.pi * n)) ** (1 / 3)
-    return radius.to(u.m)
+    return radius
 
 
-def chemical_potential(n_e: u.m ** -3, T: u.K):
+# TODO: review doctests...before decoration wrong units [cm^3] where being passed and not converted
+@validate_quantities(n_e={'can_be_negative': False},
+                     T={'can_be_negative': False,
+                        'equivalencies': u.temperature_energy()})
+def chemical_potential(n_e: u.m ** -3, T: u.K) -> u.dimensionless_unscaled:
```

Comment:
There is an issue with the `chemical_potential()` tests (`pytest`'s and `doctests`'s).  This function was not originally decorated with `check_quantity` or `validate_quantity` to handle input/output arguments.  The `n_e` argument was/is being used with units of `u.cm ** -3` while the internals are functioning like the expected units are `u.m ** -3`.